assembly/build.gradle

```diff
@@ -30,9 +30,6 @@ shadowJar {
     mergeServiceFiles()
 
     relocate 'javassist', 'ai.h2o.javassist'
-    // Relocate Eclipse Jetty to avoid collisions with Spark Jetty (newer version)
-    relocate 'org.eclipse.jetty', 'ai.h2o.org.eclipse.jetty'
```

Comment:
Please can you do the same in `benchmarks/build.gradle`?

Reply:
Good catch!